internal/pkg/template/diff/iter.go

```diff
@@ -76,6 +78,14 @@ func (i *lcsStateMachine) action() action {
 	return action
 }
 
+func (i *lcsStateMachine) peek() action {
```

Comment:
nit: (for all receivers)
```suggestion
func (sm *lcsStateMachine) peek() action {
```